src/answers-headless.ts

```diff
@@ -27,108 +27,247 @@ import answersUtilities from './answers-utilities';
 import { SelectableFilter } from './models/utils/selectablefilter';
 import { transformFiltersToCoreFormat } from './utils/transform-filters';
 
+/**
+ * Provides methods for updating the state and for sending requests to the Answers
+ * API through {@link AnswersCore}.
+ */
 export default class AnswersHeadless {
+  /**
+   * Common utility functions for manipulating Answers-related data.
+   */
   public readonly utilities = answersUtilities;
 
+  /**
+   * Sets up the AnswersHeadless instance with managers for its state and network
+   * requests and an instance of the {@link AnswersCore} library.
+   *
+   * @param core - The Answers Core instance used for sending network requests
+   * @param stateManager - The manager of the state and how it can be manipulated
+   * @param httpManager - The manager for network requests and reponses through core
+   */
```

Comment:
This constructor is actually internal since users of the library will be able to get a Headless instance through the `provideAnswersHeadless` function.  If you take a look at the `AnswersCore` class, you'll see we mark it as internal: https://github.com/yext/answers-core/blob/master/src/AnswersCore.ts

Reply:
Ah ok, I'll remove these docs then